wsclient/src/test/java/org/hpccsystems/ws/client/platform/EclParseRegressionTest.java

```diff
@@ -0,0 +1,194 @@
+package org.hpccsystems.ws.client.platform;
+
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.hpccsystems.ws.client.HPCCWsDFUClient;
+import org.hpccsystems.ws.client.HPCCWsWorkUnitsClient;
+import org.hpccsystems.ws.client.ManualUnitTest;
+import org.hpccsystems.ws.client.platform.DFUFileDetailInfo;
+import org.hpccsystems.ws.client.platform.DFULogicalFileInfo;
+import org.hpccsystems.ws.client.platform.EclRecordInfo;
+import org.hpccsystems.ws.client.platform.Platform;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+@Category(ManualUnitTest.class)
+public class EclParseRegressionTest  {
+
+    Logger logger=Logger.getLogger(this.getClass().getName());
+
+    boolean reset=false;
+
+    Platform p=null;
+    
+
+    File failedlist=new File("failedrecs.txt");    
+    File testlist=new File("alreadytested.txt");
+    List<String> alreadytested=new ArrayList<String>();
+    List<String> failedrecs=new ArrayList<String>();
+    public String getIP() {
+        return "10.241.100.159";

```

Comment:
we prob shouldn't have a hardcodded IP here. Can we default to 'localhost' and provide a mechanism to configure this value?

Reply:
For hipie we have a BaseHpccConnection test with all the test logic, then a RampsDevConnectionTest and a FourWayConnectionTest that overwrite the getIp(), etc.  Are you OK with me doing that here?

Reply:
sounds reasonable, keep in mind everything merged into this repo is public, and we tend to avoid ips, usernames, credentials,etc.